client/state/data-layer/wpcom/sites/media/index.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Internal dependencies
+ */
+import { MEDIA_REQUEST } from 'state/action-types';
+import { isRequestingMedia } from 'state/selectors';
+import { failMediaRequest, receiveMedia, requestingMedia } from 'state/media/actions';
+import wpcom from 'lib/wp';
+import debug from 'debug';
+
+/**
+ * Module variables
+ */
+const log = debug( 'calypso:middleware-media' );
+
+/**
+ * Issues an API request to fetch media for a site and query.
+ *
+ * @param  {Object}  store  Redux store
+ * @param  {Object}  action Action object
+ * @return {Promise}        Promise
```

Comment:
this is one thing that bothers me about linters - they tend to push us towards caring about the wrong things. the linter says, "nothing is there" so we put something there, but the intention of the linter was to motivate us to provide a helpful description.

my personal habits tend to be a little more aggressive against linters in this regard, where I might ignore its noise or try and fix the linting strategy.

in this case, even adding a single word to it, like "_Action_ object" helps a lot, even more so potentially something like, "Redux action"